include/glow/Importer/ONNXModelLoader.h

```diff
@@ -53,6 +53,55 @@ class ONNXModelLoader
   /// ONNX model op_version;
   size_t opsetVersion_;
 
+  // Load Constant ONNX operator.
+  llvm::Error loadConstant(const ONNX_NAMESPACE::NodeProto &op,
+                           const ArgumentDictionaryTy &dict);
+
+  // Load Slice ONNX operator.
+  llvm::Error loadSlice(const ONNX_NAMESPACE::NodeProto &op,
+                        const ArgumentDictionaryTy &dict);
+
+  // Load Conv ONNX operator.
+  llvm::Error loadConv(const ONNX_NAMESPACE::NodeProto &op,
+                       const ArgumentDictionaryTy &dict);
+
+  // Load MaxPool or AveragePool ONNX operator.
```

Comment:
can you add comments on the params which are not super obvious, like typeName here and in the rest of the methods where it's applicable.